interpolation/Dockerfile

```diff
@@ -0,0 +1,60 @@
+# base image
+FROM pelias/baseimage
+
+# libpostal apt dependencies
+# note: this is done in one command in order to keep down the size of intermediate containers
+RUN apt-get update && \
+    apt-get install -y autoconf automake libtool pkg-config time && \
+    rm -rf /var/lib/apt/lists/*
+
+# install libpostal
+RUN git clone https://github.com/openvenues/libpostal /code/libpostal
+WORKDIR /code/libpostal
+RUN ./bootstrap.sh && \
+    ./configure --datadir=/usr/share/libpostal && \
+    make && make check && make install && \
+    ldconfig
+
+# dependencies
+RUN apt-get update && \
+    apt-get install -y python sqlite3 gdal-bin lftp unzip pigz && \
+    rm -rf /var/lib/apt/lists/*
+
+# --- pbf2json ---
+
+# location of the openstreetmap data
+ENV PBF2JSON_FILE '/data/openstreetmap/portland_oregon.osm.pbf'
```

Comment:
I was using the convention `/data/openstreetmap/extract.osm.pbf`, then when you download your portland extract you can just rename it `extract.osm.pbf`.